### DISCUSSION TITLE
negative importance and package randomForestSRC

### DISCUSSION THREAD

**Discussion by JJCCLL** [kogalur/randomForestSRC#304] [Q&A]
I am working on survival random forest on a dataset using package 
randomForestSRC. We have 71 predictors. We got the variable importance 
list and found that there is negative importance, see below for some of 
them. Could anyone please help us interpret the negative importance? I 
googled and couldn't find the answer. Many thanks.

CVDHISTORY | -0.387097496
RIGHTFUGFUSIFORMGYRUS_VR | -0.405789685
CEREBELLUM_VR | -0.406030867
NUMBEROFDRUGS | -0.554088997

**Comment by ishwaran** [comment]
Generally speaking this means the model would be better off with that variable, since the prediction error is improved without it.

See this vignette: https://www.randomforestsrc.org/articles/vimp.html

**Reply by JJCCLL** [reply]
> Generally speaking this means the model would be better off with that variable, since the prediction error is improved without it.
> 
> See this vignette: https://www.randomforestsrc.org/articles/vimp.html
Thank you very much, ishwaran.

**Reply by lskatz** [reply]
I am seeing some negative numbers in my VIMP too. Do I understand your explanation correctly that negative VIMPs indicate factors that are much more important than positive VIMPs?

**Comment by ishwaran** [comment]
Negative VIMP identifies noisy variables, large positive VIMP identifies strong signal variables.

https://www.randomforestsrc.org/articles/rfsrc-subsample.html

**Reply by lskatz** [reply]
Thank you! This makes a lot more sense to me now!
